site/content/docs/developing/storage.md

```diff
@@ -0,0 +1,219 @@
+      path: {{ path at which to mount }} # Required.
+      read_only: {{ boolean }}           # Default: true
+      efs:
+        id: {{ filesystem ID }}          # Required.
+        root_dir: {{ filesystem root }}  # Optional. Defaults to "/". Must not be 
+                                         # specified if using access points.
+        auth: 
+          iam: {{ boolean }}             # Optional. Whether to use IAM authorization when 
+                                         # mounting this filesystem.
+          access_point_id: {{ access point ID}} # Optional. The ID of the EFS Access Point
+                                                # to use when mounting this filesystem.
+```
+
+### Creating Mount Targets
+There are several ways to create mount targets for an existing EFS filesystem: [using the AWS CLI](#with-the-aws-cli) and [using Addons](#with-addons).
+
+#### With the AWS CLI
+To create mount targets for an existing filesystem, you'll need 
+
+1. the ID of that filesystem.
+2. a Copilot environment deployed in the same account and region.
+
+To retrieve the filesystem ID, you can use the AWS CLI:
+```bash
+$ EFS_FILESYSTEMS=$(aws efs describe-file-systems | \
+  jq '.FileSystems[] | {ID: .FileSystemId, CreationTime: .CreationTime, Size: .SizeInBytes.Value}')
+```
+
+If you `echo` this variable you should be able to find which filesystem you need. Assign it to the variable `$EFS_ID` and continue.
+
+You'll also need the public subnets of the Copilot environment and the Environment Security Group. This jq command will filter the output of the describe-stacks call down to simply the desired output value. 
+
+!!!info
+    The filesystem you use MUST be in the same region as your Copilot environment!
+
+```bash
+$ SUBNETS=$(aws cloudformation describe-stacks --stack-name ${YOUR_APP}-${YOUR_ENV} \
+  | jq '.Stacks[] | .Outputs[] | select(.OutputKey == "PublicSubnets") | .OutputValue')
+$ SUBNET1=$(echo $SUBNETS | jq -r 'split(",") | .[0]')
+$ SUBNET2=$(echo $SUBNETS | jq -r 'split(",") | .[1]')
+$ ENV_SG=$(aws cloudformation describe-stacks --stack-name ${YOUR_APP}-${YOUR_ENV} \
+  | jq -r '.Stacks[] | .Outputs[] | select(.OutputKey == "EnvironmentSecurityGroup") | .OutputValue')
+```
+
+Once you have these, creating mount targets is simple. 
+```bash
+$ MOUNT_TARGET_1_ID=$(aws efs create-mount-target \
+    --subnet-id $SUBNET_1 \
+    --security-groups $ENV_SG \
+    --file-system-id $EFS_ID | jq -r .MountTargetID)
+$ MOUNT_TARGET_2_ID=$(aws efs create-mount-target \
+    --subnet-id $SUBNET_2 \
+    --security-groups $ENV_SG \
+    --file-system-id $EFS_ID | jq -r .MountTargetID)
+```
+
+Once you've done this, you can specify the `storage` configuration in the manifest as above. 
+
+##### Cleanup
+
+Delete the mount targets using the AWS CLI. 
+
+```bash
+$ aws efs delete-mount-target --mount-target-id $MOUNT_TARGET_1
+$ aws efs delete-mount-target --mount-target-id $MOUNT_TARGET_2
+```
+
+#### With Addons
```

Comment:
I wonder if instead of addons, we should just say `CloudFormation` and not go through the hack of creating a job.

Something like:
```
If you'd like to create a EFS FS from scratch, you can use the create a new CloudFormation stack using the following template:
```